src/analyses/relationAnalysis.ml

```diff
@@ -0,0 +1,651 @@
+open Prelude.Ana
+open Analyses
+open RelationDomain
+
+module M = Messages
+
+module SpecFunctor (CPriv: RelationPriv.S) (RD: RelationDomain.RD) (PCU: RelationPrecCompareUtil.Util) =
+struct
+  include Analyses.DefaultSpec
+
+  module Priv = CPriv (RD)
+  module D = RelationDomain.RelComponents (RD) (Priv.D)
+  module G = Priv.G
+  module C = D
+  module V = 
+  struct
+    include Priv.V
+    include StdV
+  end
+
+  module RV = RD.V
+
+  module PCU = PCU(RD)
+
+  (* Result map used for comparison of results *)
+  let results = PCU.RH.create 103 (*ToDO Remove this*)
```

Comment:
Remove.

Reply:
Actually we have this on master without such TODO for `exp.apron.prec-dump` for dumping octagons for some traces-rel comparison. @michael-schwarz, will this actually still be needed or can it be removed after all?

Reply:
I would vote to leave these facilities (`exp.apron.prec-dump`) there, at least until the corresponding paper is accepted. After that we can remove it, if it really hinders maintenance. Otherwise it is nice to have in case we want to do a follow-up paper.